readthedocs/projects/tasks.py

```diff
@@ -183,12 +183,15 @@ def run(self, version_pk):  # pylint: disable=arguments-differ
             self.project = self.version.project
             self.sync_repo()
             return True
-        # Catch unhandled errors when syncing
+        except RepositoryError as e:
+            # Do not log as ERROR handled exceptions
+            log.warning('There was an error with the repository: msg=%s', e.msg)
```

Comment:
We could pass `exc_info=True` to `log.warning` if you want the full exception.

Reply:
You are right. Changing.